package.json

```diff
@@ -58,7 +58,7 @@
     "sia-import": "(test -d Sia || mkdir -p Sia) && (cp $GOPATH/bin/siad Sia & cp $GOPATH/bin/siac Sia)",
     "sia-kill": "killall siad; while pgrep siad; do sleep .5; done;",
     "sia-refresh": "npm run sia-kill && rm -rf Sia && npm run sia-import",
-    "release-args": "webpack && electron-packager . Sia-UI --app-version=1.0.1 --version=1.2.2 --overwrite --prune --ignore='(node_modules|release|config.json|errors.log)' --out=release/v1.0.1 --icon=assets/icon",
+    "release-args": "webpack && electron-packager . Sia-UI --app-version=1.0.2 --overwrite --prune --ignore='(node_modules|release|config.json|errors.log)' --out=release/v1.0.2 --icon=assets/icon",
```

Comment:
what is the effect of removing the `--version` arg? Will it just use the most recent version?